src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -66,62 +67,58 @@ export default class MxGraphRenderer {
       const bounds = shape.bounds;
       const parent = this.getParent(bpmnElement);
       const absoluteCoordinate = { x: bounds.x, y: bounds.y };
-      const style = this.computeStyleName(shape);
+      const style = this.computeStyle(shape);
       this.insertVertexGivenAbsoluteCoordinates(parent, bpmnElement.id, bpmnElement.name, absoluteCoordinate, bounds.width, bounds.height, style);
     }
   }
 
-  private computeStyleName(shape: Shape): string {
-    const bpmnElement = shape.bpmnElement;
+  private computeStyle(bpmnCell: Shape | Edge): string {
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    const styleValues = new Map<string, any>();
 
-    let style = bpmnElement.kind as string;
-    if (bpmnElement instanceof ShapeBpmnEvent) {
-      style += ';' + StyleConstant.BPMN_STYLE_EVENT_KIND + '=' + bpmnElement.eventKind;
+    const font = bpmnCell.label?.font;
+    if (font) {
+      styleValues.set(this.mxConstants.STYLE_FONTFAMILY, font.name);
+      styleValues.set(this.mxConstants.STYLE_FONTSIZE, font.size);
+      styleValues.set(this.mxConstants.STYLE_FONTSTYLE, this.getFontStyleValue(font));
     }
-    style += this.computeFontStyle(shape);
 
-    return style;
-  }
-
-  private computeFontStyle(bpmnCell: Shape | Edge): string {
-    let style = '';
+    const bpmnElement = bpmnCell.bpmnElement;
+    if (bpmnElement instanceof ShapeBpmnEvent) {
+      styleValues.set(StyleConstant.BPMN_STYLE_EVENT_KIND, bpmnElement.eventKind);
+    }
 
-    const font = bpmnCell?.label?.font;
-    if (font) {
-      if (font.name) {
-        style += ';' + this.mxConstants.STYLE_FONTFAMILY + '=' + font.name;
+    let style = bpmnElement.kind as string;
+    styleValues.forEach((value, key) => {
+      if (value) {
+        style += ';' + key + '=' + value;
       }
+    });
 
-      if (font.size) {
-        style += ';' + this.mxConstants.STYLE_FONTSIZE + '=' + font.size;
-      }
+    return style;
+  }
 
-      if (font.isBold) {
-        style += ';' + this.mxConstants.STYLE_FONTSTYLE + '=' + this.mxConstants.FONT_BOLD;
-      } else if (font.isItalic) {
-        style += ';' + this.mxConstants.STYLE_FONTSTYLE + '=' + this.mxConstants.FONT_ITALIC;
-      } else if (font.isStrikeThrough) {
-        style += ';' + this.mxConstants.STYLE_FONTSTYLE + '=' + this.mxConstants.FONT_STRIKETHROUGH;
-      } else if (font.isUnderline) {
-        style += ';' + this.mxConstants.STYLE_FONTSTYLE + '=' + this.mxConstants.FONT_UNDERLINE;
-      }
+  private getFontStyleValue(font: Font): number {
```

Comment:
:question: I suggest to add unit test to increase coverage on this method as e2e tests don't cover all paths
I can handle that